### ISSUE TITLE
Mean dice coming out to be 0

### ISSUE THREAD

**Issue by DecentMakeover** [Tencent/MedicalNet#2]
Hey i have trained Medical Net on custom data, but even though the loss approaches 0,When i test the mean dice for every class is 0(there are 3 total classes).

But this result is only after i trained for 4 epochs.

Any suggestions?

Thanks in advance.

**Comment by cshwhale** [comment]
Hi, what's the type of task for your custom data? Segmentation, detection or classification?

**Comment by DecentMakeover** [comment]
Hi, thanks for the response

Its a 2 class segmentation task.

**Comment by cshwhale** [comment]
Hi, there are 3 suggestions:
1. If your custom data is small amount, it may be better to use batchnorm parameters from the pre-trained model during training. 
2. If the task is unbalanced segmentation (small-sized target), patch-based segmentation method may be a good choice.
3. Use shallow neural network or skip-connection.
Hope it helps.

**Comment by DecentMakeover** [comment]
Thanks for your suggestions, Ill check .
